oclif.manifest.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "4.6.3",
+  "version": "4.6.4",
```

Comment:
is this cause the automated release doesn't update this?

Reply:
Yeah, it must be. This was auto-updated by oclif on build